### ISSUE TITLE
Central Media-Upload

### ISSUE THREAD

**Issue by Rothwild** [bueltge/multisite-global-media#41]
Is it possible to upload all media directly to a subdomain? This plugin doesn't solve the problem, does it?

I have a multisite (example.com) with different subpages (xy1.example.com, xy2.example.com) and a central media subdomain (media.example.com). What works well with this plugin is to download the media from example.com or xy1.example.com from media.example.com. If I want to upload pictures under xy1.example.com, they are unfortunately not linked/displayed at media.example.com. And I cannot embed them at xy2.example.com. 

Can you give me a hint here?

**Comment by bueltge** [comment]
Yes. This plugin creates a media area thats need a separate site of the Multisite. It is also possible to use a subdomain or separate domain on default in the Multisite instance of WordPress.

**Comment by Rothwild** [comment]
Thanks a lot for your answer!

Yes, i have a separate site (subdomain) in my multisite network only for media - but the plugin does NOT force the upload on any subdomain to media.example.com  ...  xy1.example.com loads the files to xy1.example.com, xy2.example.com to xy2.example.com and an uploaded file on xy1.example.com or xy2.example.com is NOT visible on media.example.com:  If I want to upload pictures under xy1.example.com, they are unfortunately not linked/displayed at media.example.com. And I cannot embed them at xy2.example.com.

Can you give me a hint here again?

**Comment by bueltge** [comment]
You can only upload media files to the global media library inside the Site, there you have defined for this job. The idea is, that peoples upload media files central, in each site, to use it in other site.

**Comment by Rothwild** [comment]
Thank you very much for your answer. Do you have an workaround or idea how I can prohibit users from uploading media to the site that will use the Media Center's images?

**Comment by bueltge** [comment]
@Rothwild please read #54 - the right idea, solution?
